src/main/java/core/basesyntax/service/parse/DataParserImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.parse;
+
+import core.basesyntax.model.FruitRecord;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataParserImpl implements DataParser {
+
+    @Override
+    public List<FruitRecord> parseFruitRecords(List<String> lines) {
+        List<FruitRecord> transactions = new ArrayList<>();
+
+        for (int i = 1; i < lines.size(); i++) {
+            String line = lines.get(i);
+            String[] parts = line.split(",");
```

Comment:
Consider defining a constant for the delimiter "," used in the split operation to avoid magic strings and improve maintainability.